pilot/cmd/pilot-agent/main.go

```diff
@@ -586,7 +588,16 @@ func getMeshConfig() (meshconfig.MeshConfig, error) {
 		}
 		return *mc, nil
 	}
-	return defaultConfig, nil
+	b, err := ioutil.ReadFile(meshConfigFile)
```

Comment:
Can you please add comments on the order of precedence - MESH_CONFIG will be attempted first and then `meshConfigFile` etc?

Reply:
@howardjohn I understand you meant to remove the `MESH_CONFIG ` eventually